tangelo/linq/target/backend.py

```diff
@@ -255,16 +255,16 @@ def simulate(self, source_circuit, return_statevector=False, initial_statevector
 
         if desired_meas_result is not None:
             if not isinstance(desired_meas_result, str) or len(desired_meas_result) != n_meas:
-                raise ValueError("desired_meas result is not a string with the same length as the number of measurements"
+                raise ValueError("desired_meas result is not a string with the same length as the number of measurements "
```

Comment:
Just to be sure: we are adding spaces here because of the continuation comment, because it currently prints the last word of a line attached to the first word of the next, and it actually needs a space to separate them ?

Reply:
Yes, it was missing a space. When concatenating like that, the spaces must be explicit.